maintainer-groups.yaml

```diff
@@ -0,0 +1,73 @@
+maintainerGroups:
+- name: Helm Org
+  ownersLink: https://github.com/helm/community/blob/main/MAINTAINERS.md
+  repos:
+  - https://github.com/helm/community
+  - https://github.com/helm/pull-sizer
+  - https://github.com/helm/charts-tooling
+  - https://github.com/helm/specs
+  - https://github.com/helm/query-store-quay-logs
+  - https://github.com/helm/github-webhook-dco-labeler
+  - https://github.com/helm/charts-check-pr-title
+  - https://github.com/helm/repo-audit
+  - https://github.com/helm/examples
+  mailingList: cncf-helm-core-maintainers@lists.cncf.io
```

Comment:
The mailing list is `cncf-helm-org-maintainers@lists.cncf.io`. Can you please update it